From khali@linux-fr.org Mon Oct 17 13:53:17 2005
Date: Mon, 17 Oct 2005 22:53:52 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 03/14] Owner field additions to many i2c drivers, 3 of 5
Message-Id: <20051017225352.708f8c24.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-owner-field-03-i2c-keywest.patch

From: Laurent Riffard <laurent.riffard@free.fr>

This patch updates the .owner field for various struct xxxx_driver
variables which are available on PPC_MAC arch.

This one was _not_ even compile-tested...

Signed-off-by: Laurent Riffard <laurent.riffard@free.fr>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/i2c/busses/i2c-keywest.c |    2 ++
 1 file changed, 2 insertions(+)

--- gregkh-2.6.orig/drivers/i2c/busses/i2c-keywest.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-keywest.c
@@ -716,6 +716,7 @@ static struct of_device_id i2c_keywest_m
 
 static struct macio_driver i2c_keywest_macio_driver = 
 {
+	.owner		= THIS_MODULE,
 	.name 		= "i2c-keywest",
 	.match_table	= i2c_keywest_match,
 	.probe		= create_iface_macio,
@@ -724,6 +725,7 @@ static struct macio_driver i2c_keywest_m
 
 static struct of_platform_driver i2c_keywest_of_platform_driver = 
 {
+	.owner		= THIS_MODULE,
 	.name 		= "i2c-keywest",
 	.match_table	= i2c_keywest_match,
 	.probe		= create_iface_of_platform,
